moto/ec2/models.py

```diff
@@ -3222,6 +3223,25 @@ def create_vpc_endpoint(
 
         return vpc_end_point
 
+    def get_vpc_end_point(self, vpc_end_point_ids, filters=None):
+        vpc_end_points = self.vpc_end_points.values()
+
+        if vpc_end_point_ids:
+            vpc_end_point_ids = [
+                vpc_end_point
+                for vpc_end_point in vpc_end_points
+                if vpc_end_point.id in vpc_end_point_ids
+            ]
+            if len(vpc_end_points) != len(vpc_end_point_ids):
+                invalid_id = list(
+                    set(vpc_end_point_ids).difference(
+                        set([vpc_end_point.id for vpc_end_point in vpc_end_points])
+                    )
+                )[0]
+                raise InvalidVpcEndPointIdError(invalid_id)
```

Comment:
Hi @usmangani1, thanks for your PR! Would you mind adding a unit test for this error scenario?
Happy to merge after that.